main.py

```diff
@@ -213,11 +354,12 @@ def zenodo(name, g, results):
             html = gr.HTML("<h1 style = 'font-size: 20px;'>Search Results</h1>")
     search.click(sources, input_text, html)
 
+
 def open_alex(name, g, results):
-    serializable_results, results = find(name, results)
+    serializable_results = search_openalex.find(name, results)
     search_result = json.dumps(serializable_results)
+
     g.parse(data=search_result, format='json-ld')
-    return g, results
 
 
-demo.launch()
+demo.launch(server_name="0.0.0.0")
```

Comment:
Maybe there's a reason but why was this changed? It raising an error on my local PC but I guess it works fine on the Docker environment?

Reply:
Oh, does it? I added it because without this, Gradio would only listen on 127.0.0.1, i.e. localhost. Then it would be of no use inside a Docker container as it cannot be accessed from the outside.

Reply:
I see! If it works on the Docker then all cool 😎